oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/web/reactive/function/client/OAuth2AuthorizedClientResolver.java

```diff
@@ -70,6 +70,7 @@ public OAuth2AuthorizedClientResolver(
 	 * If true, a default {@link OAuth2AuthorizedClient} can be discovered from the current Authentication. It is
 	 * recommended to be cautious with this feature since all HTTP requests will receive the access token if it can be
 	 * resolved from the current Authentication.
+	 *
```

Comment:
There are no code changes in this file - only formatting changes. Or am I missing something?

Reply:
I don't remember adding a blank line between the text description in the comment & the @param line, but I see I did...removed now. (sheepish grin)